bookwyrm/templates/snippets/status/status_header.html

```diff
@@ -1,72 +1,94 @@
 {% load bookwyrm_tags %}
 {% load i18n %}
-<span
-    itemprop="author"
-    itemscope
-    itemtype="https://schema.org/Person"
->
-    <a
-        href="{{ status.user.local_path }}"
-        itemprop="url"
-    >
-        {% include 'snippets/avatar.html' with user=status.user ariaHide="true" %}
+{% load humanize %}
 
-        <span itemprop="name">{{ status.user.display_name }}</span>
-    </a>
-</span>
+<div class="media">
+    <figure class="media-left">
+        <a class="image is-48x48" href="{{ status.user.local_path }}">
+            {% include 'snippets/avatar.html' with user=status.user ariaHide="true" medium="true" %}
+        </a>    
+    </figure>
```

Comment:
Moving the avatar here means moving it out of of the _Person_ scope, which might not be ideal. We’re also duplicating links.

Reply:
What could be done to affect the image to the _Person_ scope?

Reply:
(This is a summary of our chat on [matrix])

In the not so distant past, I couldn’t manage to make the microdata `itemref` to work consistently.

In our case, I would probably end up using this kind of solution:

```html
{% with user=status.user %}
    <div class="media">
        <figure class="media-left" aria-hidden="true">
            <a class="image is-48x48" href="{{ user.local_path }}">
                {% include 'snippets/avatar.html' with medium="true" %}
            </a>
        </figure>

        <div class="media-content">
            <h3 class="has-text-weight-bold">
                <span
                    itemprop="author"
                    itemscope
                    itemtype="https://schema.org/Person"
                >
                    {% if user.avatar %}
                        <meta itemprop="image" content="/images/{{ user.avatar }}">
                    {% endif %}

                    <a
                        href="{{ user.local_path }}"
                        itemprop="url"
                    >
                        <span itemprop="name">{{ user.display_name }}</span>
                    </a>
                </span>

                […]
            </h3>
        </div>
    </div>
{% endwith %}
```
Changes in this code are:

- abstracting `status.user` with a `{% with user=status.user %}` to reduce code a bit and declare an intention for the block of code.
- moving the `aria-hidden` from the `img` to the `figure` to avoid redundancy of contiguous links with different labels leading to the same URL.
- adding the `<meta>` to keep the image property within its scope. This also means we can serve a bigger image for the avatar to clients using microdata, and a smaller one for people with visual capabilities.

I’ll let you pick what you want from my proposal, is any. :)

Reply:
Changes will be coming later today :)